client/components/domains/trademark-claims-notice/index.jsx

```diff
@@ -0,0 +1,563 @@
+						{ this.renderTrademarkClaimMark( claim ) }
+						{ this.renderTrademarkClaimJurisdiction( claim ) }
+						{ this.renderTrademarkClaimGoodsAndServices( claim ) }
+						{ this.renderTrademarkClaimInternationalClassification( claim ) }
+						{ this.renderTrademarkClaimRegistrant( claim ) }
+						{ this.renderTrademarkClaimContact( claim ) }
+						{ this.renderTrademarkClaimCases( claim ) }
+					</li>
+				) ) }
+			</ol>
+		);
+	};
+
+	// Note: Please don't update the language rendered here as it is specified in the RPM-Requirements of the
+	// ICANN TMCH functional specifications in: https://tools.ietf.org/html/draft-ietf-eppext-tmch-func-spec-01#ref-RPM-Requirements
+	renderNotice() {
+		return (
+			<Fragment>
+				<CompactCard className="trademark-claims-notice__content">
+					<h3>Trademark Notice</h3>
+					<p>
+						You have received this Trademark Notice because you have applied for a domain name which
+						matches at least one trademark record submitted to the Trademark Clearinghouse.
+					</p>
+					<p>
+						<strong>
+							<em>
+								You may or may not be entitled to register the domain name depending on your
+								intended use and whether it is the same or significantly overlaps with the
+								trademarks listed below. Your rights to register this domain name may or may not be
+								protected as noncommercial use or “fair use” by the laws of your country.
+							</em>
+						</strong>
+					</p>
+					<p>
+						Please read the trademark information below carefully, including the trademarks,
+						jurisdictions, and goods and services for which the trademarks are registered. Please be
+						aware that not all jurisdictions review trademark applications closely, so some of the
+						trademark information below may exist in a national or regional registry which does not
+						conduct a thorough or substantive review of trademark rights prior to registration. If
+						you have questions, you may want to consult an attorney or legal expert on trademarks
+						and intellectual property for guidance.
+					</p>
+					<p>
+						If you continue with this registration, you represent that, you have received and you
+						understand this notice and to the best of your knowledge, your registration and use of
+						the requested domain name will not infringe on the trademark rights listed below. The
+						following marks are listed in the Trademark Clearinghouse:
+					</p>
+					{ this.renderTrademarkClaimsInfo() }
+				</CompactCard>
+				{ this.renderNoticeActions() }
+			</Fragment>
+		);
+	}
+
+	render() {
+		const { isSignupStep } = this.props;
+		const { showFullNotice } = this.state;
+
+		return (
+			<div className="trademark-claims-notice">
+				{ ! isSignupStep && this.renderHeader() }
+				<div className="trademark-claims-notice__content">
+					{ this.renderPreamble() }
+					{ ! showFullNotice && this.renderShowNoticeLink() }
+					{ showFullNotice && this.renderNotice() }
```

Comment:
Might be a bit easier to read if you combine these two lines and just write `{ showFullNotice ? this.renderNotice() : this.renderShowNoticeLink() }`? Also ZOMG like 10 characters saved!!!!1111111ONEONEONOEE!!!